### PR TITLE
CASMHMS-5736 Turn CAPMC power on/off/reinit into a shim that calls PCS

### PR DESCRIPTION
**Pull request by mjendrysik-hpe**
### Summary and Scope

Convert the CAPMC xname_[on/off/reinit] and get_xname_status APIs into shims that call PCS.

### Issues and Related PRs

* Resolves [CASMHMS-5736](https://jira-pro.its.hpecorp.net:8443/browse/CASMHMS-5736)

### Testing

Tested on:

* unit tests
* Tavern tests with simulated environment

Were the install/upgrade based validation checks/tests run?(goss tests/install-validation doc) N
Were continuous integration tests run? Y/N  Y
Was an Upgrade tested? N - not tested on hardware yet
Was a Downgrade tested? N - not tested on hardware yet

Verified that all valid unit and tavern tests passed. Some unit tests were removed due to CAPMC not needing to do certain behaviors anymore.

### Risks and Mitigations

HAS A SECURITY AUDIT BEEN RUN? Y
ARE THERE KNOWN ISSUES WITH THESE CHANGES? N
ANY OTHER SPECIAL CONSIDERATIONS? Y

Requires:

* Additional testing on a CSM based Shasta Olympus system

### CODE REVIEW

**Comment by nieuwsma** [comment]
need to bump version numbers 😄

**Comment by mjendrysik-hpe** [comment]
> need to bump version numbers 😄

Yeah yeah yeah. If you notice the PR is in Draft mode.

I had no idea what new checks I was going to run into.

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ Fixed - for any bug fixes
 Security - in case of vulnerabilities
 -->
 
+## [3.0.0] - 2022-01-03
```
Comment (schooler-hpe) [inline CHANGELOG.md:27]: The year/date could use an update here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ Fixed - for any bug fixes
...
+
+### Changed
+
+- Converted xname_[on/off/reinit] to use PCS for power operations
```
Comment (schooler-hpe) [inline CHANGELOG.md:31]: get_xname_status as well.

**Review by schooler-hpe — APPROVED** [review]

File: docker-compose.test.ct.yaml [hunk 3]
```diff
@@ -333,13 +370,15 @@ services:
       - LOG_LEVEL=DEBUG
       - TRS_IMPLEMENTATION=LOCAL
       - HSMLOCK_ENABLED=true
+      - PCS_URL=http://cray-power-control:28007/testing
```
Comment (nieuwsma) [inline docker-compose.test.ct.yaml:373]: what is this URL? /testing??

**Review by nieuwsma — CHANGES_REQUESTED** [review]

**Review by nieuwsma — APPROVED** [review]

**Review by alvarez3-hpe — APPROVED** [review]

**Pull request merged**